lib/ember-orbit/schema.js

```diff
@@ -151,13 +151,13 @@ var Schema = Ember.Object.extend({
           var rel = record.__rel[link] = {};
           linkValue.forEach(function(id) {
             if (typeof id === 'object') {
-              id = id.primaryId;
+              id = id.get('primaryId');
             }
             rel[id] = true;
           });
 
         } else if (typeof linkValue === 'object') {
-          record.__rel[link] = linkValue.primaryId;
+          record.__rel[link] = linkValue.get('primaryId');
```

Comment:
I've seen that pattern a lot but have never understood what the benefit is. Is it a question of style or is there a practical benefit? (will certainly make the change, I've just always been curious about this).